### ISSUE TITLE
Yocto 2.5

### ISSUE THREAD

**Issue by francisduhaut** [TechNexion/edm-yocto-bsp#14]
Hello Richard

Is a release done with Yocto 2.5 ?
We want to move to QT5.12 and Yocto 2.4 is linked to QT5.9

Thanks

**Comment by zahabshams** [comment]
Is it going to support pico nymph configuration

**Comment by richard-hu** [comment]
@francisduhaut 
@zahabshams 

Yes. We have plan to upgrade to Yocto 2.5 for all TechNexion products.

BR,

Richard
